### DISCUSSION TITLE
Unknown plugin option bug

### DISCUSSION THREAD

**Discussion by rubenesque-code** [gatsbyjs/gatsby#31465] [Help] [answered]
## Description

I'm using `gatsby-plugin-image` with `gatsby-plugin-sharp`. With the default option `layout` set for `gatsby-plugin-sharp`, when running `gatsby develop`, I get a warning:  

> there are unknown plugin options for "gatsby-plugin-sharp": defaults.layout

I haven't made a query for the image data yet. My config file:

```js
module.exports = {
  plugins: [
    `gatsby-plugin-emotion`,
    `gatsby-plugin-image`,
    {
      resolve: `gatsby-plugin-sharp`,
      options: {
        defaults: {
          formats: [`auto`, `webp`, `avif`],
          layout: `fullWidth`,
          quality: 80,
        },
      },
    },
    `gatsby-transformer-sharp`,
    {
      resolve: `gatsby-source-strapi`,
      options: {
        apiURL: `xxx.herokuapp.com`,
        queryLimit: 1000,
        contentTypes: [
          `products`,
        ],
      },
    },
  ],
}
```

### Steps to reproduce
I don't have a minimal reproduction up sorry but can get one up if needed.

### Expected result
I expect to be able to set a default layout in the config file.

### Environment

  System:
    OS: Linux 5.8 Ubuntu 20.04.2 LTS (Focal Fossa)
    CPU: (4) x64 Intel(R) Core(TM) i5-7400 CPU @ 3.00GHz
    Shell: 5.8 - /usr/bin/zsh
  Binaries:
    Node: 12.18.3 - ~/.nvm/versions/node/v12.18.3/bin/node
    npm: 6.14.10 - ~/.nvm/versions/node/v12.18.3/bin/npm
  Browsers:
    Chrome: 90.0.4430.212
    Firefox: 88.0.1
  npmPackages:
    gatsby: ^3.4.1 => 3.4.1
    gatsby-plugin-emotion: ^6.5.0 => 6.5.0
    gatsby-plugin-eslint: ^3.0.0 => 3.0.0
    gatsby-plugin-image: ^1.5.0 => 1.5.0
    gatsby-plugin-sharp: ^3.5.0 => 3.5.0
    gatsby-source-filesystem: ^3.5.0 => 3.5.0
    gatsby-source-shopify: ^4.5.0 => 4.5.0
    gatsby-source-strapi: ^1.0.0-alpha.0 => 1.0.0-alpha.0
    gatsby-transformer-sharp: ^3.5.0 => 3.5.0
  npmGlobalPackages:
    gatsby-cli: 3.5.0

**Comment by LekoArts** [comment, marked as answer]
You can't set a default `layout` for every image query. It'll be `CONSTRAINED` by default, if you want to change that you'll need to do that in the individual query / StaticImage component
